lib/solid_queue/app_executor.rb

```diff
@@ -4,7 +4,7 @@ module SolidQueue
   module AppExecutor
     def wrap_in_app_executor(&block)
       if SolidQueue.app_executor
-        SolidQueue.app_executor.wrap(&block)
+        SolidQueue.app_executor.wrap(source: "application.solid_queue", &block)
```

Comment:
> application.solid_queue

Not sure if these values are the best🙃 

I followed the naming rules described [here](https://github.com/rails/rails/pull/44999#discussion_r873140565).

Reply:
That seems reasonable to me, reading that comment there 🤔